include/dnscpp/lookup.h

```diff
@@ -95,6 +103,11 @@ class Lookup : public Operation
      *  @return bool        was a result reported to userspace?
      */
     virtual bool execute(double now) = 0;
+
```

Comment:
I think that this docblock is no longer accurate, and the name of the function is now too vague now that the semantics are changed.